packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
+
+### Outputs
+
+None
+
+### Events
+
+- @change: Fires whenever the slider's value is changed
+
+### Slots
+
+- default: the primary slot intended for the `input` element
+- thumb: the draggable element of the slider
+- lower-thumb: same as `thumb`, but used in `range` mode
+- upper-thumb: same as `thumb`, but used in `range` mode
+- track: the slider's base and used to display the min/max selectable values
+- progress-track: the progress indicator when in `range` mode
+
+### CSS Variables
+
+None
+
+### Interactions
+
+#### thumb
+
+- On :hover a tooltip appears indicating the current value of the slider
+
+#### track
+
+- Clicking on the track moves the thumb to the location of mouse click
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/slider/
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="slider" in FAST
+  - [x] aria-valuenow in FAST
+  - [x] aria-valuemin in FAST
+  - [x] aria-valuemax in FAST
+  - [x] aria-orientation (if vertical) in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [ ] Up / Right : Increments slider by Step amount
+  - [ ] Down / Left: Decrements slider by Step amount
+  - [ ] PageUp/Up/Right & Shift : Increments the value of the slider by 10 \_ step.
+  - [ ] PageDown /Down/Left & Shift : Decrements the value of the slider by 10 \_ step.
+  - [ ] Home : Sets value to the min prop.
+  - [ ] End : Sets value to the max prop.
+
+## Preparation
+
+- [x] [Find the base FAST Component](https://explore.fast.design/components/) this component will inherit from and document
+  - [FAST Slider Component](https://explore.fast.design/components/fast-slider)
+- [x] [Check the Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/tree/master/specs) for differences and document
+  - [Fluent Slider Spec](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-slider/docs/Spec.md)
+  - no `defaultValue` attribute, instead only `value` that specifies the initial selected range of the slider
+  - `rail` is in Fluent React; `track` is in FAST Foundation
+  - Current implementation in Fluent React 9 does not have `range`, but Fast Foundation Slider and Figma design spec has it
```

Comment:
If I understand the Figma spec correctly, the `range` there means the value indicators before and after the slider.

Reply:
Not that they wouldn't need to implement the labels, just that the SliderLabel is a separate component. Could probably word this better.